raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -339,26 +366,48 @@ private VoteResponseData handleVoteRequest(VoteRequestData request) throws IOExc
         }
     }
 
-    private void handleVoteResponse(int remoteNodeId, VoteResponseData response) throws IOException {
-        if (quorum.isLeader()) {
-            logger.debug("Ignoring vote response {} since we already became leader for epoch {}",
+    /**
+     * Handle a received Vote response.
```

Comment:
True, this is mostly filler since I decided to add the javadoc. Since these methods are all following the same convention, perhaps there is no need to document each one.